pkg/autoscaler/statserver/server.go

```diff
@@ -195,7 +195,6 @@ func (s *Server) Shutdown(timeout time.Duration) {
 	case <-time.After(time.Until(shutdownStart.Add(timeout))):
```

Comment:
I think this should be using `<-ctx.Done()` right?

Reply:
Done